process/sync/baseSync.go

```diff
@@ -130,16 +132,15 @@ type baseBootstrap struct {
 	isInImportMode               bool
 	scheduledTxsExecutionHandler process.ScheduledTxsExecutionHandler
 	processWaitTime              time.Duration
+	processWaitTimeSupernova     time.Duration
+	preparedForSync              bool
 
 	repopulateTokensSupplies bool
 }
 
-// TODO: remove this handling after async exec
 func (boot *baseBootstrap) getProcessWaitTime() time.Duration {
 	if boot.enableRoundsHandler.IsFlagEnabled(common.SupernovaRoundFlag) {
```

Comment:
i think we should check here if enabled by round; where we call this, we have access to header, so we can check by round

Reply:
pushed